terraform/slurm_cluster/modules/slurm_controller_instance/variables.tf

```diff
@@ -282,6 +282,15 @@ EOD
   default = []
 }
 
+variable "enable_external_prolog_epilog" {
+  description = <<EOD
+Automatically enable a script that will execute prolog and epilog scripts
+shared under /opt/apps from the controller to compute nodes.
```

Comment:
I would like to the example from the variable description.

Reply:
I believe this was done in PR #50. Let me know if I'm misinterpreting.